### PR TITLE
[avmfritz] Use system channel types and decrease minimum polling interval

### LINKED ISSUE #13655: [avmfritz] pollingInterval and HAN_FUN_CONTACT
In my opinion, pollingInterval minimum has to be lower than 5. It's to high for door/window contact. In addition, it would be nice to have a channel linked to "lastalertchgtimestamp" (when the timestamp change, it could mean that someone opened the door/window and be detected even with a long pollingInterval).

https://github.com/openhab/openhab-addons/blob/1e07d1af03fff4410b84d950479b1474595fa1e9/bundles/org.openhab.binding.avmfritz/src/main/java/org/openhab/binding/avmfritz/internal/handler/AVMFritzBaseBridgeHandler.java#L135

### PR DESCRIPTION
**Pull request by cweitkamp**
- Use system default channel types
- Changed minimum polling interval to 1s

Closes #13655

Signed-off-by: Christoph Weitkamp <github@christophweitkamp.de>

### CODE REVIEW

File: bundles/org.openhab.binding.avmfritz/src/main/java/org/openhab/binding/avmfritz/internal/handler/Powerline546EHandler.java [hunk 1]
```diff
@@ -195,11 +217,9 @@ private void updateThingChannelState(String channelId, State state) {
...
         ThingHandlerCallback callback = getCallback();
         if (callback != null) {
-            ChannelUID channelUID = new ChannelUID(thing.getUID(), channelId);
-            ChannelTypeUID channelTypeUID = CHANNEL_BATTERY.equals(channelId)
-                    ? DefaultSystemChannelTypeProvider.SYSTEM_CHANNEL_BATTERY_LEVEL.getUID()
-                    : new ChannelTypeUID(BINDING_ID, channelId);
-            Channel channel = callback.createChannelBuilder(channelUID, channelTypeUID).build();
+            final ChannelUID channelUID = new ChannelUID(thing.getUID(), channelId);
+            final ChannelTypeUID channelTypeUID = createChannelTypeUID(channelId);
+            final Channel channel = callback.createChannelBuilder(channelUID, channelTypeUID).build();
```
Comment (jlaur) [inline bundles/org.openhab.binding.avmfritz/src/main/java/org/openhab/binding/avmfritz/internal/handler/Powerline546EHandler.java:222]: Consider labels and descriptions (e.g. "Voltage" -> "Electric Voltage").

**Review by jlaur — CHANGES_REQUESTED** [review]
Thanks, looks good. I have added four minor comments for consideration regarding the changed labels and descriptions.

File: bundles/org.openhab.binding.avmfritz/src/main/resources/OH-INF/thing/thing-types.xml [hunk 2]
```diff
@@ -91,7 +91,10 @@
 			<channel id="locked" typeId="locked"/>
 			<channel id="device_locked" typeId="device_locked"/>
 			<channel id="temperature" typeId="temperature"/>
-			<channel id="actual_temp" typeId="actual_temp"/>
+			<channel id="actual_temp" typeId="system.indoor-temperature">
+				<label>Current Temperature</label>
+				<description>Current measured temperature.</description>
+			</channel>
```
Comment (J-N-K) [inline bundles/org.openhab.binding.avmfritz/src/main/resources/OH-INF/thing/thing-types.xml:97]: If you change the channel-type, please add update instructions for existing things.

Comment (cweitkamp) [reply bundles/org.openhab.binding.avmfritz/src/main/resources/OH-INF/thing/thing-types.xml:97]: They are already part of this PR. Or did I miss one of the changed Things.

Comment (jlaur) [reply bundles/org.openhab.binding.avmfritz/src/main/resources/OH-INF/thing/thing-types.xml:97]: And I already forgot that I reviewed them. :flushed:

Comment (cweitkamp) [reply bundles/org.openhab.binding.avmfritz/src/main/resources/OH-INF/thing/thing-types.xml:97]: By the way: I missed to change the channel types in the https://github.com/openhab/openhab-addons/blob/4fb55d3350dfb3767abf1e1f6b5dfa220be05fc7/bundles/org.openhab.binding.avmfritz/src/main/resources/OH-INF/thing/bridge-types.xml. Will do that.

Do the update instructions work for bridge-types too? I am not able to add tag `<bridge-types>` in the `instructions.xml`.

Comment (J-N-K) [reply bundles/org.openhab.binding.avmfritz/src/main/resources/OH-INF/thing/thing-types.xml:97]: `thing-type` should also work for bridges.

File: bundles/org.openhab.binding.avmfritz/src/main/java/org/openhab/binding/avmfritz/internal/handler/Powerline546EHandler.java (hunk 1 above)
Comment (jlaur) [reply bundles/org.openhab.binding.avmfritz/src/main/java/org/openhab/binding/avmfritz/internal/handler/Powerline546EHandler.java:222]: @cweitkamp - just checking, I think label was changed from "Voltage" (and description "Current voltage") to "Electric Voltage" here. Maybe it's okay?

Comment (cweitkamp) [reply bundles/org.openhab.binding.avmfritz/src/main/java/org/openhab/binding/avmfritz/internal/handler/Powerline546EHandler.java:222]: Yes, that is fine.

**Review by jlaur — APPROVED** [review]
LGTM

**Comment by helmar74** [comment]
Hi, is there any possibility to test it upfront or do I have to wait until OH 4.0.0 Milestone 5?

**Pull request merged**